### ISSUE TITLE
[High Contrast] Menu: Hovered items are hard to read

### ISSUE THREAD

**Issue by rigdern** [winjs/winjs#1399]
1. Open `menu.html` from http://1drv.ms/1LlLARL in Edge
2. Switch the machine to use "High Contrast White"
3. Click the `show menu` button to open the menu
4. Hover over the command labeled `button option`.

**Unexpected result:** The hovered command is very hard to read because it's black text on a purple background. Instead, the text should be white.

![image](https://cloud.githubusercontent.com/assets/199935/9398746/6d7ad47e-475f-11e5-940f-bc8d799fc82d.png)
